enterprise/coderd/oauth2.go

```diff
@@ -208,10 +208,14 @@ func (api *API) postOAuth2ProviderAppSecret(rw http.ResponseWriter, r *http.Requ
 		})
 		return
 	}
+	// TODO: Currently unused.
+	prefix, _ := cryptorand.String(40)
```

Comment:
Basically a stop-gap, this is replaced in the next PR.  I figure once this is approved I merged that one here, and then merge both into main together.